storage/innobase/xtrabackup/doc/source/xtrabackup_bin/xbk_option_reference.rst

```diff
@@ -155,6 +153,9 @@ Options
    unless :option:`--force-non-empty-directories` option is
    specified.
 
+   Any tables detected as missing are not removed automatically. Remove such
```

Comment:
this warning is shown when we start the server, not during the copy-back. Can we moved them to a better place?